packages/@magic-ext/aptos/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018-present Magic Labs, Inc.
```

Comment:
?
```suggestion
Copyright (c) 2023-present Magic Labs, Inc.
```

Reply:
yes? it should be 2023

Reply:
fixed it
[e04b3d9](https://github.com/magiclabs/magic-js/pull/515/commits/e04b3d9549eb33400556c531b453a587122d0102)

Reply:
https://github.com/magiclabs/magic-js/blob/master/LICENSE
Actually, we changed the LICENSE recently, could you update it? 

I think in a later patch, we should update all LICENSE in the repo and the scaffold script